src/mmw/js/src/modeling/controllers.js

```diff
@@ -35,22 +36,84 @@ var ModelingController = {
                 name: 'Untitled Project',
                 created_at: Date.now(),
                 area_of_interest: App.map.get('areaOfInterest'),
-                scenarios: new models.ScenariosCollection([
-                    new models.ScenarioModel({
-                        name: 'Current Conditions',
-                        is_current_conditions: true,
-                        active: true
-                    })
-                ])
+                scenarios: new models.ScenariosCollection()
+            });
+
+            // Poll for model results and set the results in the scenario when done.
+            var pollModel = function(scenario) {
```

Comment:
I think we may need to re-architect this a bit to make it testable since it's a crucial component of the app. What is now `pollModel` could be reworked as a method on `ScenarioModel`.